test/test-meshctl-ls.json

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
Im using `test-meshctl-ls.json`,  `test-meshctl-ls.render-full` and `test-meshctl-ls.render-2` and sample files for testing purposes. The actual dependencies don't matter.